gui/qt/seed_dialog.py

```diff
@@ -100,6 +101,9 @@ def __init__(self, seed=None, title=None, icon=True, msg=None, options=None, is_
             self.is_seed = is_seed
             self.saved_is_seed = self.is_seed
             self.seed_e.textChanged.connect(self.on_edit)
+            self.suggestion = SeedSuggestion("./lib/wordlist/english.txt")
```

Comment:
Not sure if this will work in the binaries.

`lib` will be installed as `electrum` so the path might be different.